assessmentModel/src/commonMain/kotlin/survey/QuestionState.kt

```diff
@@ -147,7 +157,11 @@ abstract class AbstractQuestionFieldStateImpl : QuestionFieldState {
             // `JsonNull` is used as a special placeholder for "chose not to answer".
             currentResult.jsonValue == JsonNull ||
```

Comment:
FYI: this is a hold-over from design wanting "prefer not to answer" as a selectable response that is *different* from "skip" where "skip" is `null` and "prefer not to answer" is `JsonNull` while running the survey. (To allow for saving answers during back-forth navigation).